site/package.json

```
@@ -67,8 +67,8 @@
         "@babel/preset-env": "^7.20.2",
         "@babel/preset-react": "^7.18.6",
         "@babel/preset-typescript": "^7.21.0",
-        "@testing-library/react": "^14.0.0",
-        "@testing-library/user-event": "^14.4.3",
+        "@testing-library/react": "^14.3.1",
```

```suggestion
        "@testing-library/react": "^16.0.0",
```

latest version is 16 :)

---

done